java/src/main/native/src/CuFileJni.cpp

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cufile.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <memory>
+
+#include <stdexcept>
```

Comment:
Why are there blank lines between standard library headers? Typically we group those together and alphabetize them. Maybe check that clang-format is being applied. It may be the case that we don’t have include groupings in the clang-format settings for this repo.

Reply:
My local formatter must have not worked on this file. I have updated it